bower.json

```diff
@@ -7,6 +7,7 @@
     "blueimp-canvas-to-blob": "2.1.0",
     "chai": "1.8.1",
     "cocktail": "0.5.9",
+    "fxa-password-strength-checker": "https://github.com/Mozilla/fxa-password-strength-checker.git#master",
```

Comment:
ixnay on `#master` We want a definitive git sha or a version number, otherwise changes to the fxa-password-strength-checker could inadvertently break the content server.